api.go

```diff
@@ -8,26 +8,26 @@ import (
 
 	"github.com/maniksurtani/quotaservice/config"
 	"github.com/maniksurtani/quotaservice/logging"
-
-	pbconfig "github.com/maniksurtani/quotaservice/protos/config"
+	pb "github.com/maniksurtani/quotaservice/protos/config"
 )
 
 // The Server interface is what you get when you create a new quotaservice.
 type Server interface {
 	Start() (bool, error)
 	Stop() (bool, error)
 	SetLogger(logger logging.Logger)
-	ServeAdminConsole(*http.ServeMux, string, config.ConfigPersister, bool)
+	CreateBucketContainer(*pb.ServiceConfig)
```

Comment:
Yeah, it could be part of Start(). It just seems overly difficult to create optional defaults, so I left it as a separate method and initialized the default in Start().